app/Models/RockTheVoteReport.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Chompy\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class RockTheVoteReport extends Model
+{
+    /**
+     * We use the externally created Rock the Vote ID as our primary key.
+     */
+    public $incrementing = false;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'id',
+        'status',
+        'since',
+        'before',
+        'dispatched_at',
+    ];
+
+    /**
+     * The attributes that should be mutated to dates.
+     *
+     * @var array
+     */
+    protected $dates = [
+        'since',
+        'before',
+        'dispatched_at',
+    ];
+
+    /**
+     * Attributes that can be queried when filtering.
+     *
+     * This array is manually maintained. It does not necessarily mean that
+     * any of these are actual indexes on the database... but they should be!
+     *
+     * @var array
+     */
+    public static $indexes = [
+        'status',
+        'dispatched_at',
+    ];
```

Comment:
I'm not so sure this is correct, or necessary. In the migration, I added a compound index on both `status` and `dispatched_at` to find any reports that are complete but not yet dispatched.

Also started a discussion in [Slack](https://dosomething.slack.com/archives/CUQMTP0RM/p1583779493003800?thread_ts=1583776494.002800&cid=CUQMTP0RM) about whether to rely on a job to keep checking the status of a report (vs a scheduled task that queries our DB)